### PR TITLE
E2E test logs output error info

### PR DESCRIPTION
**Pull request by qiuming-best**
Signed-off-by: Ming <mqiu@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change
Most of the e2e test error info is not output into logs, It's not easy to troubleshoot problems, so adjust the test logs, refer to #4580
# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: test/e2e/backups/sync_backups.go [hunk 1]
```diff
@@ -54,88 +54,92 @@ func (b *SyncBackups) Init() {
...
-			Expect(VeleroInstall(context.Background(), &VeleroCfg, "", false)).To(Succeed())
+			err = VeleroInstall(context.Background(), &VeleroCfg, "", false)
+			Expect(err).To(Succeed(), "Failed to install velero with error %v", err)
 		}
 	})
 
 	AfterEach(func() {
 		if VeleroCfg.InstallVelero {
-			Expect(VeleroUninstall(context.Background(), VeleroCfg.VeleroCLI, VeleroCfg.VeleroNamespace)).To(Succeed())
+			VeleroUninstall(context.Background(), VeleroCfg.VeleroCLI, VeleroCfg.VeleroNamespace)
```
Comment (blackpiglet) [inline test/e2e/backups/sync_backups.go:69]: Seems should add "err = " at front of this line.

File: test/e2e/privilegesmgmt/ssr.go [hunk 2]
```diff
@@ -45,84 +45,91 @@ func SSRTest() {
...
-			fmt.Sprintf("Failed to create %s namespace", testNS))
+		By(fmt.Sprintf("Create %s namespace", testNS), func() {
+			err = CreateNamespace(ctx, client, testNS)
+			Expect(err).To(Succeed(), "Failed to create %s namespace with error %v", testNS, err)
+		})
+
+		By(fmt.Sprintf("Get version in %s namespace", testNS), func() {
+			cmd := []string{"version", "-n", testNS}
+			err = VeleroCmdExec(context.Background(), VeleroCfg.VeleroCLI, cmd)
+			Expect(err).To(Succeed(), "Failed to create an ssr object in the %s namespace with error %v", testNS, err)
```
Comment (blackpiglet) [inline test/e2e/privilegesmgmt/ssr.go:74]: Log content is not right.

File: test/e2e/privilegesmgmt/ssr.go [hunk 3]
```diff
@@ -45,84 +45,91 @@ func SSRTest() {
...
+		})
+
+		By(fmt.Sprintf("Get version in %s namespace", VeleroCfg.VeleroNamespace), func() {
```
Comment (blackpiglet) [inline test/e2e/privilegesmgmt/ssr.go:77]: Seems this block is duplicated with the above.

File: test/e2e/backups/sync_backups.go (hunk 1 above)
Comment (qiuming-best) [reply test/e2e/backups/sync_backups.go]: OK， I've modified the code

File: test/e2e/privilegesmgmt/ssr.go (hunk 2 above)
Comment (qiuming-best) [reply test/e2e/privilegesmgmt/ssr.go:74]: if we execute command `velero version -n testNS`, which will create an ssr cr in testNS namespace by the controller

**Pull request closed without merging**